oap-server/server-storage-plugin/storage-elasticsearch-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/elasticsearch/base/IndexController.java

```diff
@@ -132,6 +146,11 @@ public static class LogicIndicesRegister {
          */
         public static final String RECORD_TABLE_NAME = "record_table";
 
+        /**
+         * The management data table name in aggregation physical storage.
```

Comment:
```suggestion
         * The management data table name in merged physical storage.
```

Do we choose the term `merge` or `aggregation`? I would say `merged-index mode` is a better term. Could you align those in the docs and code comments?

Reply:
agree